vochain/indexer/migrations/0003_alter_table_processes_votes.sql

```diff
@@ -1,5 +1,6 @@
 -- +goose Up
 ALTER TABLE vote_references ADD voter_id BLOB NOT NULL DEFAULT X'';
+ALTER TABLE vote_references ADD overwrite_count INTEGER NOT NULL;
```

Comment:
you should add a new migration with a higher number in the filename, otherwise sqlite databases which are up to date will not get the new column.

you also need to add a default, given the NOT NULL, e.g. `DEFAULT 0`.

you should also add the "down" variant.

Reply:
cc @jordipainan since we recently investigated why a dvote node was failing to upgrade the db, given we had modified an existing migration once it hit master.

Reply:
yup, and this should be added after #700

Reply:
> you should add a new migration with a higher number in the filename, otherwise sqlite databases which are up to date will not get the new column.

Since we don't need backwards compatibility at this point, I'd take profit of it and remove all migrations. This is the reason why I did not create a new file and just used an existing one. However, that should be done in a separated commit.

> you also need to add a default, given the NOT NULL, e.g. DEFAULT 0.

In theory this is already addressed by Golang since an uint32 has the default value 0. But probably its more correct to add it also here, so I'll do. Thanks.

> you should also add the "down" variant.

What's that?

Reply:
You do need a default, because these are sql values, not go :)

see the "Down" bits at the bottom of the file.